include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -385,9 +396,32 @@ Parameters::check(dealii::ConditionalOStream const & pcout) const
     }
   }
 
+  if(spatial_discretization == SpatialDiscretization::HDIV)
+  {
+    AssertThrow(
+      use_divergence_penalty == false || use_continuity_penalty == false,
+      dealii::ExcMessage(
+        "The use of penalty terms does not make sense in the case of HDIV since they are evaluated to zero. They should therefore be deactivated."));
+    if(temporal_discretization != TemporalDiscretization::BDFCoupledSolution)
```

Comment:
Why "!=" and not "=="?

Please introduce brackets for this if branch (it is formally not needed but a coding convention in ExaDG).